cli/cmd/top.go

```diff
@@ -614,9 +660,11 @@ func stripPort(address string) string {
 	return strings.Split(address, ":")[0]
 }
 
-func (t *topTable) renderHeaders() {
+func (t *topTable) renderHeaders(scrollXpos int) {
 	tbprint(0, 0, "(press q to quit)")
-	x := 0
+	tbprint(0, 1, "(press a/LeftArrowKey to scroll left, d/RightArrowKey to scroll right)")
```

Comment:
I think you need to increase the `headerHeight` constant by 1 to accommodate for this keep the blank line separating the header from the column headers.

Reply:
Done